client/hooks/useForm.js

```diff
@@ -5,7 +5,7 @@ import { capitalize } from '../helpers/capitalize';
 
 const getValue = (e) => (e.currentTarget ? e.currentTarget.value : e);
 
-export const useForm = (obj) => {
+export const useForm = (obj, onChange = () => {}) => {
```

Comment:
I think `client/hooks/useForm.js` was accidentally merged, as it was replaced by `client/hooks/useForm.ts`.